src/components/Search/Result.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import { Text, StyleSheet, View, Pressable, PressableProps, ViewStyle } from 'react-native'
+
+import Colours from '../../themes/colours'
+
+const styles = StyleSheet.create( {
+  container: {
+    padding: 5,
+    backgroundColor: Colours.MediumGray,
+    borderBottomColor: Colours.LightGray,
+    borderBottomWidth: 1,
+  },
+  header: {
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+  headerText: {
+    fontSize: 15,
+    fontWeight: '400',
+  },
+  line: {
+    marginBottom: 5,
+    fontSize: 22,
+    fontWeight: '600',
+  },
+  translation: {
+    fontSize: 15,
+  },
+} )
+
+export type SearchResultDataProps = {
+  /**
+   * Name of gurbani source
+   */
+  source: string,
+  /**
+   * Page number in source
+   */
+  page: string,
+  /**
+   * Last accessed date
+   */
+  date?: string,
+  /**
+   * Gurbani Line
+   */
+  line: string,
+  /**
+   * Translation for gurbani line
+   */
+  translation: string,
+}
+
+export type SearchResultProps = {
+  /**
+   * Style for `View` container
+   */
+  customStyle?: ViewStyle,
+  onPress: NonNullable<PressableProps['onPress']>,
+} & Omit<PressableProps, 'style'> & SearchResultDataProps
+
+const SearchResult = ( {
+  source,
+  page,
+  date,
+  line,
+  translation,
+  onPress,
+  customStyle,
```

Comment:
```suggestion
  style,
```
etc?

Reply:
I did start with style but it wasn't working for some reason.

Reply:
Give it a go again

Reply:
tried didn't work :(

Reply:
It should definitely work - what doesn't work when you name it such?

Reply:
They style doesn't show up

Reply:
Commit it and I'll help ya debug. Have used this convention in the other work I've been doing so far.

Reply:
I don't know but it works now :)